projects/packages/my-jetpack/_inc/components/product-cards-section/style.module.scss

```
@@ -66,6 +66,17 @@
 	line-height: var(--font-title-small);
 }
 
+.my-jetpack-products-container {
+	padding-left: 0;
+	padding-right: 0;
```

Although it doesn't matter here as it's the same padding on both the sides, but RTL aware alternative for this would be a single line

```suggestion
	padding-inline: 0;
```